WorkbookApps/Xamarin.Workbooks.Mac/Build.proj

```diff
@@ -6,8 +6,10 @@
     </PropertyGroup>
     <MSBuild Projects="BuildBundle.proj" Targets="$(Targets)" Properties="WorkbookAppTarget=Modern;LSMinimumSystemVersion=10.11" />
     <MSBuild Projects="BuildBundle.proj" Targets="$(Targets)" Properties="WorkbookAppTarget=Modern;LSMinimumSystemVersion=10.12" />
+    <MSBuild Projects="BuildBundle.proj" Targets="$(Targets)" Properties="WorkbookAppTarget=Modern;LSMinimumSystemVersion=10.13" />
     <MSBuild Projects="BuildBundle.proj" Targets="$(Targets)" Properties="WorkbookAppTarget=Full;LSMinimumSystemVersion=10.11" />
     <MSBuild Projects="BuildBundle.proj" Targets="$(Targets)" Properties="WorkbookAppTarget=Full;LSMinimumSystemVersion=10.12" />
+    <MSBuild Projects="BuildBundle.proj" Targets="$(Targets)" Properties="WorkbookAppTarget=Full;LSMinimumSystemVersion=10.13" />
```

Comment:
Since this is for 1.5, can we drop one of the other bundles now?

Reply:
I was going to see how much bigger the installer became first.